addons/advanced_ballistics/CfgVehicles.hpp

```diff
@@ -1,4 +1,16 @@
 class CfgVehicles {
+    class Man;
+    class CAManBase: Man {
+        class ACE_SelfActions {
+            class ACE_CheckAirTemperature {
+                displayName = CSTRING(CheckAirTemperature);
+                condition = QUOTE(GVAR(enabled));
+                statement = QUOTE([_player] call FUNC(getApproximateAirTemp));
+                icon = QPATHTOEF(overheating,UI\temp_ca.paa);
```

Comment:
this does create a requirement between ab and overheating
might want to copy icon over to ab